docs/user-guide/data-structures/creating-arrays-and-datasets.ipynb

```diff
@@ -244,11 +323,17 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
    "source": [
     "`arange` does not include the stop value but `linspace` does by default.\n",
     "Please note that the caveats described in [NumPy's documentation](https://numpy.org/doc/stable/user/basics.creation.html#d-array-creation-functions) apply to Scipp as well.\n",
     "\n",
+    "All range-like functions currently use NumPy to generate values and thus incur the same costs from copying as `scipp.array`.\n",
```

Comment:
Didn't you add a comment above saying that cost can be avoided by using these? I suppose this only refers to zeros and ones?